.ci/Jenkinsfile

```diff
@@ -150,7 +151,11 @@ pipeline {
               }
               post {
                 always {
-                  junit(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/outputs/TEST-unit-*.xml")
+                  withEnv([
+                      "JUNIT_OTEL_TRACE_NAME=junit-unit-tests"
+                  ]){
+                    junit2otel(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/outputs/TEST-unit-*.xml")
```

Comment:
I'm just wondering if the signature for this step could be changed to use arguments instead env variables? Probably we have discussed this in the past, sorry if so. No blocking but just an idea

Reply:
Yes we did :) https://github.com/elastic/apm-pipeline-library/pull/1514/commits. In particular,  https://github.com/elastic/apm-pipeline-library/pull/1514/commits/ed989e9ffd6a6508cf1e5d8ae12fc6fad93f9fbc we decided to go with env vars

Reply:
<img width="1162" alt="Screenshot 2022-03-30 at 11 08 58" src="https://user-images.githubusercontent.com/951580/160795453-aae8e595-b493-481a-ae3c-064389c971af.png">

Reply:
Gotcha, so the signature is the main issue, I now remember what about using a map argument and then remove it when calling the junit (args)

```
call(def args) {
   otelArgs = args.otelArgs
   def service = otelArgs.service
   ...
   ...   

   return junit(args.delete(otelArgs))
```

Sorry for asking something not relevant to this particular PR, but somehow I'm thinking about the usability and whether it makes sense before rolling out to different places the new step

Reply:
indeed! for me the simplest usage would be:

```
junit2otel(traceName: "junit-unit-tests", allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/outputs/TEST-unit-*.xml")
```

Which I think we can achieve it now, because we started overriding `junit` but we renamed the step to `junit2otel`. So I guess we can use params instead.

wdyt if we put this in the back burner and rewrite the junit2otel step to accept parameters?

Reply:
> wdyt if we put this in the back burner and rewrite the junit2otel step to accept parameters?

Totally agree :)

Reply:
I submitted the changes in https://github.com/elastic/apm-pipeline-library/pull/1629

Reply:
I changed the signature to use the step properly